src/pages/categories.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+// import {  useDispatch } from 'react-redux'
+// import { status } from '../redux/categories/categoriesSlice';
```

Comment:
[OPTIONAL]

- [ ] I kindly suggest you remove commented-out code from your codebase as it doesn't make it look neat.